### ISSUE TITLE
searchParameters no longer supporting clickAnalytics?

### ISSUE THREAD

**Issue by ondrejmirtes** [algolia/docsearch#2397]
I just updated from 3.6.2 to 3.8.0 and I get this error:

> Error: src/js/MainMenuViewModel.ts(36,4): error TS2322: Type '{ clickAnalytics: true; }' is not assignable to type 'SearchForHits | undefined'.
  Type '{ clickAnalytics: true; }' is not assignable to type 'SearchParamsQuery & BaseSearchParamsWithoutQuery & IndexSettingsAsSearchParams & { indexName: string; type?: "default" | undefined; } & { ...; }'.

For the following piece of code:

```
		docsearch({
			container: '#docsearch',
			appId: '563YUB35R3',
			indexName: 'phpstan',
			apiKey: '38f6379285feb01cc915c6967c715ec2',
			searchParameters: {
				clickAnalytics: true,
			},
		});
```

On 3.6.2 the clickAnalytics parameter was known. This is a backward compatibility break.

How should I fix the code? Thanks.

**Comment by randombeeper** [comment]
@millotp could this be related to recent changes?

**Comment by millotp** [comment]
Hello, thanks for reporting the issue, indeed the type for `searchParameters` was not correct in 3.8.0, this is fixed in 3.8.1
